common/src/main/java/org/candlepin/common/exceptions/mappers/BadQueryParameterHandler.java

```diff
@@ -0,0 +1,97 @@
+package org.candlepin.common.exceptions.mappers;
+
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import javax.inject.Provider;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.ResponseBuilder;
+import javax.ws.rs.core.Response.Status;
+
+import org.apache.commons.lang.StringUtils;
+import org.candlepin.common.exceptions.CandlepinParameterParseException;
+import org.candlepin.common.exceptions.ExceptionMessage;
+import org.candlepin.common.util.VersionUtil;
+import org.xnap.commons.i18n.I18n;
+
+import com.google.inject.Inject;
+
+/**
+ * Class is able to detect whether an exception contains error related to wrong
+ * parameters. In addition to that this class is able to build an http error response
+ * for that exception.
+ * 
+ * Historical notes: This had to be extracted from BadRequestExceptionMapper, because the new
+ * RestEasy changed the way it handles (now throws NotFoundException)
+ * unparseable parameters of JAX-RS resources.
+ * 
+ * @author fnguyen
+ *
+ */
+public class BadQueryParameterHandler {//extends CandlepinExceptionMapper {
```

Comment:
If this no longer extends CandlepinExceptionMapper then this comment should be removed